src/plugins/data/common/search/aggs/date_interval_utils/parse_interval.ts

```diff
@@ -49,6 +49,13 @@ export function parseInterval(interval: string): moment.Duration | null {
       u => Math.abs(duration.as(u)) >= 1
     ) as unitOfTime.Base;
```

Comment:
Nit: The typing here can be simplified - then no typecasts and no anys are needed:
```ts
const selectedUnit = find(
      dateMath.units,
      u => Math.abs(duration.as(u)) >= 1
    )!;

if (dateMath.units.indexOf(selectedUnit) < dateMath.units.indexOf(unit)) {
      return duration;
    }

```